ports/geckolib/glue.rs

```diff
@@ -3811,8 +3811,7 @@ impl<'a> PrioritizedPropertyIter<'a> {
         let mut sorted_property_indices: Vec<PropertyAndIndex> =
             properties.iter().enumerate().map(|(index, pair)| {
                 PropertyAndIndex {
-                    property: PropertyId::from_nscsspropertyid(pair.mProperty)
-                              .unwrap_or(all.clone()),
+                    property: PropertyId::from_nscsspropertyid(pair.mProperty).unwrap_or(all.clone()),
```

Comment:
It's not actually important because that clone is a no op, but maybe we should define  `all` in the loop and avoid the clone, because it looked expensive to me before I checked.